test/data/settlements/{settlementId}/participants/{participantId}/accounts/{accountId}.js

```diff
@@ -0,0 +1,191 @@
+/*****
```

Comment:
Why are you are re-adding the mocked tests that you removed on the last PR?

Reply:
It is required in src/handlers/settlements/{settlementId}/participants/{participantId}/accounts/{accountId}.js